static/js/contributors.js

```diff
@@ -0,0 +1,44 @@
+function insertContributors() {
```

Comment:
Can the code of this js file be placed in `useEffect` of `src/pages/team/index.js`, so that there is no need for an extra link to load js?